GetBrokerRedirectURI.ps1

```diff
@@ -0,0 +1,103 @@
+<#
+.SYNOPSIS
+Obtain the RedirectURI from an APK.  Requires JDK\bin\keytool and AndroidSDK\build-tools\###\aapt to be on the path.
+
+.EXAMPLE
+GetBrokerRedirectURI MyApp.apk
+#>
+Param(
+[Parameter(Mandatory=$true)] 
+[string]$apkFile
+)
+
+<#
+Extract all of the signature owner and SHA1 hex bytes from `keytool -printcert -jarfile`.
+#>
+function GetSHA1Hex {
+    Param(
+        [string]$apkFile
+        )
+    $output = keytool -printcert -jarfile $apkFile
```

Comment:
Same comment here relative to use of a command.  Verify it's available.

Reply:
see other thread